kratos/sources/kratos_parameters.cpp

```diff
@@ -358,6 +358,9 @@ void Parameters::AddValue(
 {
     if(mpValue->find(rEntry) == mpValue->end()) {
         (*mpValue)[rEntry] = *(rOtherValue.mpValue);
+    } else {
+        KRATOS_WARNING("Parameters") << "WARNING:: Entry " << rEntry << " already defined. Overriding it" << std::endl;
+        SetValue(rEntry, rEntry);
```

Comment:
Shouldn't this be
```suggestion
        SetValue(rEntry, rOtherValue);
```
?

Reply:
The message is fine.